### PR TITLE
README update: compile using the 2.0.0 version

### PR DESCRIPTION
**Pull request by oldergod**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Download
 Gradle:
 
 ```groovy
-compile 'com.squareup.whorlwind:whorlwind:1.0.1'
+compile 'com.squareup.whorlwind:whorlwind:2.0.0'
```
Comment (JakeWharton) [inline README.md]: Make this `implementation` while you're here?

**Comment by mattprecious** [comment]
I thought I replied when I merged this... Anyway. It was released in the minutes before merging 🙂

**Comment by vanniktech** [comment]
Release tag seems to be missing.

**Comment by mattprecious** [comment]
Added

**Pull request merged**